src/main/kotlin/gg/skytils/skytilsmod/core/Config.kt

```diff
@@ -2428,6 +2438,28 @@ object Config : Vigilant(
     )
     var totemPing = 0
 
+    @Property(
+        PropertyType.SWITCH, name = "Attunement Display",
+        description = "Recolors the Inferno boss and demons depending on the correct dagger attunement.",
+        category = "Slayer", subcategory = "Inferno Demonlord"
+    )
+    var attunementDisplay = false
+
+    @Property(
+        PropertyType.SWITCH, name = "Ignore Pacified Blazes",
+        description = "Stops rendering faraway blazes when fighting the Inferno Demonlord if Smoldering Polarization is active.\n" +
```

Comment:
Is this supposed to be "faraway"?

Reply:
Yes, nearby blazes aren't hidden because the player can still interact with them and also because there might be a bug with minibosses still moving towards the player but not attacking. The range at which the blazes are hidden is just 3 blocks away, but I couldn't find a more appropiate word for that distance

Reply:
I mean specifically "faraway" as opposed to "far away". Not trying to nitpick but it seems like a typo maybe?

Reply:
Both of them are correct and mean pretty much the same thing, although "faraway" is an adjective and "far away" is an adverb phrase. Could also be rephrased as "...blazes that are far away..."